src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -472,7 +481,7 @@ pub struct CompleteStreamFragmentGraph {
 }
 
 pub struct FragmentGraphUpstreamContext {
-    upstream_mview_fragments: HashMap<TableId, Fragment>,
+    upstream_job_fragments: HashMap<TableId, Fragment>,
```

Comment:
Well, `mview_fragment` means the fragments involving `Materialize` executor. `job_fragment` makes it ambiguous. What about renaming it to `root_fragments`? Same for other occurrences. cc @yezizp2012

Reply:
`root_fragments` LGTM.